Data/Scripts/012_Battle/001_Battler/006_Battler_AbilityAndItem.rb

```diff
@@ -126,8 +126,13 @@ def pbOnAbilityChanged(oldAbil)
   #=============================================================================
   # Held item consuming/removing
   #=============================================================================
-  def pbCanConsumeBerry?(_item,alwaysCheckGluttony=true)
-    return false if isUnnerved?
+  def canConsumeBerry?
+    return false if @battle.pbCheckOpposingAbility(:UNNERVE, @index)
+    return true
+  end
```

Comment:
master now does its own thing with Unnerve to condense its mentions into one place. This probably makes def isUnnerved? redundant, but I believe said method lists some Gen 8 abilities that will need to be moved into here. Make sure def isUnnerved? is no longer used and deleted, and make sure def canConsumeBerry? now does all the things it needs to.